client/src/components/ModelGraphViewerComponent/ModelGraphViewerComponent.js

```diff
@@ -302,7 +302,7 @@ class ModelGraphViewerComponent extends React.Component {
       await this.cyRef.current.doLayout(this.progressCallback);
     } else {
       this.cyRef.current.addRelationships(this.componentRelationships, "component");
-      await this.cyRef.current.doLayout(this.progressCallback);
+      await this.cyRef.current(this.progressCallback);
```

Comment:
Is this right? We can call current as a function?

Reply:
You are completly right I debugged the functionality and manage to display this error when the component was called as a function 
![image](https://user-images.githubusercontent.com/97867829/151878994-c4b4d478-fb7d-4ece-8a5c-18fe2642e603.png)